src/west/log.py

```diff
@@ -40,9 +41,23 @@ def dbg(*args, level=VERBOSE_NORMAL):
     print(*args)
 
 
-def inf(*args):
-    '''Print an informational message.'''
+def inf(*args, colorize=False):
+    '''Print an informational message.
+
+    colorize (default: False):
+      If True, the message is printed in bright green if stdout is a terminal.
+    '''
+    # This approach colorizes any sep= and end= text too, as expected.
+    #
+    # The final flush=True avoids issues with unrelated output from commands
+    # (usually Git) becoming green, due to the final attribute reset ANSI
+    # escape getting line-buffered.
+    #
+    # colorama automatically strips the ANSI escapes when stdout isn't a
+    # terminal (by wrapping sys.stdout).
+    if colorize: print(colorama.Fore.LIGHTGREEN_EX, end='')
     print(*args)
+    if colorize: print(colorama.Style.RESET_ALL, end='', flush=True)
 
 
 def wrn(*args):
```

Comment:
Can you please add a `colorize` kwarg to all of the routines, for consistency?

Some ideas for how to handle it:

- `dbg` ignores it
- `wrn` prints orange
- `err` and `die` prints red (and `die` defaults it to True)

Reply:
Went for red for both warnings and errors. Is that okay?

Saw a lot of red `WARNING:`s and `ERROR:`s when trying to un-hose my system today, so might've been inspired by that. :P